src/EnergyPlus/MixedAir.cc

```diff
@@ -4216,23 +4119,28 @@ void VentilationMechanicalProps::CalcMechVentController(
                     // Calc the breathing-zone OA flow rate
                     OAIndex = this->ZoneDesignSpecOAObjIndex(ZoneIndex);
                     if (OAIndex > 0) {
-                        {
-                            auto const SELECT_CASE_var(state.dataSize->OARequirements(OAIndex).OAFlowMethod);
-                            if (SELECT_CASE_var == OAFlowPPer) {
-                                ZoneOABZ = ZoneOAPeople;
-                            } else if (SELECT_CASE_var == OAFlow) {
-                                ZoneOABZ = ZoneOAFlow;
-                            } else if (SELECT_CASE_var == OAFlowPerArea) {
-                                ZoneOABZ = ZoneOAArea;
-                            } else if (SELECT_CASE_var == OAFlowACH) {
-                                ZoneOABZ = ZoneOAACH;
-                            } else if (SELECT_CASE_var == OAFlowSum) {
-                                ZoneOABZ = ZoneOAPeople + ZoneOAArea + ZoneOAFlow + ZoneOAACH;
-                            } else if (SELECT_CASE_var == OAFlowMax) {
-                                ZoneOABZ = max(ZoneOAPeople, ZoneOAArea, ZoneOAFlow, ZoneOAACH);
-                            } else {
-                                ZoneOABZ = 0.0;
-                            }
+                        switch (state.dataSize->OARequirements(OAIndex).OAFlowMethod) {
```

Comment:
Feels strangely familiar.

Reply:
Seeing this idiom repeatedly makes me think that `ZoneOAPeople`, `ZoneOAArea`, etc. should be replaced by `std::array<Real64, static_cast<int>(OAFlowCalcMethod::Num>` and then just accessed as `ZoneOA[static_cast<int>OAFlowMethod];` instead of these ubiquitous `switch` statements.

Reply:
Thanks for your insight. Since there is another local variable named `ZoneOA`, I'll use the name of ` ZoneOACalc`